Solutions/Ais.Net.Models/Ais/Net/Models/Abstractions/ShipTypeGroup.cs

```diff
@@ -0,0 +1,59 @@
+﻿// <copyright file="ShipTypeGroup.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Ais.Net.Models.Abstractions
+{
+    public enum ShipTypeGroup
+    {
+        NotAvailable,
```

Comment:
Do you need explicit numbering here as per the other one if this corresponds to actual numeric values? Or conversely, not in the other case?

Reply:
The point of these two new enums it to be able to roll up from the specific type to the "group / category" level, so that we can easily come up with high level grouping of vessel type (and in the notebook create colour codings)

Reply:
Sure. Why does one have explicit numbering and not the other?

Reply:
Because a specific number is meaningless when it represents a range

Reply:
Ah, so these groups *don't* correspond to a specific number in actual data; you're deriving them? Where do the groupings come from, then?

Reply:
https://help.marinetraffic.com/hc/en-us/articles/205579997-What-is-the-significance-of-the-AIS-Shiptype-number-